test/integration/util/util.go

```diff
@@ -364,8 +364,7 @@ func InitTestAPIServer(t *testing.T, nsPrefix string, admission admission.Interf
 func WaitForSchedulerCacheCleanup(sched *scheduler.Scheduler, t *testing.T) {
 	schedulerCacheIsEmpty := func() (bool, error) {
 		dump := sched.Cache.Dump()
-
-		return len(dump.Nodes) == 0 && len(dump.AssumedPods) == 0, nil
+		return (dump == nil || dump.Nodes == nil || len(dump.Nodes) == 0) && len(dump.AssumedPods) == 0, nil
```

Comment:
I don't know about such scenario, but I thought there might be so drafted the change, I will revert it to avoid confusion and minimize the diff as it is not needed